settings.gradle.kts

```diff
@@ -14,5 +14,7 @@ include(
     ":samples:adbserver-sample",
     ":samples:kaspresso-sample",
     ":samples:kautomator-sample",
-    ":samples:kautomator-sample-app-upgrade"
+    ":samples:kautomator-sample-app-upgrade",
+
+    ":tutorials"
```

Comment:
`tutorial` or `tutorials`? As I remember, this module should contain only a test app with the most common complex cases

Reply:
```tutorial``` sure, fixed